### DISCUSSION TITLE
Env. variable is not updating when using postman flows

### DISCUSSION THREAD

**Discussion by ToobaMunir** [postmanlabs/postman-flows#292] [Defect]
Hi,
I have written few test scripts in postman where I am fetching the values from API response and saving data in env. variables. When running the individual requests then the variables values are being added/ updated correctly but, when I am executing the same request using postman flows then the env. variables are not updated.
To check weather the test script is executing or not while the flow is running I added a dummy log in console.
The log in console is printed out that means the test script is running when request was executed via flow but, unable to understand that why it didn't update the env. variables.
![image](https://user-images.githubusercontent.com/81964870/161788748-d549fc74-0b42-4eed-b384-45c742f572ab.png)

`var data = pm.response.json();
pm.environment.set("itinerary_id_Bus", data.data.buses[0].itinerary_id);
pm.environment.set("lookupBus",data.data.itinerary_lookup_key)
pm.environment.set("deptDateBus",data.data.search_query.departure_date)`
console.log("I am running yayy")

**Comment by ArtsiomHilevich** [comment]
Hey @ToobaMunir , I ran into the same issue. Found an answer here https://github.com/postmanlabs/postman-flows/discussions/142

**Reply by ToobaMunir** [reply]
Hey @ArtsiomHilevich , thanks a lot. Will check this out

**Comment by shritasharma** [comment]
I am running into the same issue as well. I am unable to use the environment variables in the flow.
